storage/storage_image.go

```diff
@@ -44,6 +44,7 @@ var (
 
 type storageImageSource struct {
 	imageRef        storageReference
+	directory       string // Temporary directory where we store blobs until Commit() time
```

Comment:
Please update the comment (and, absolutely non-blocking, maybe move it below the `image` ~constant member).